pages/results.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import useForm from "../components/FormProvider";
 import Layout from "../components/Layout/Layout";
 import DownloadResults from "../components/ResultsPageComponents/DownloadResults/DownloadResults";
+import SurveyInfo from "../components/ResultsPageComponents/SurveyInfo/SurveyInfo";
```

Comment:
Hi @OlaShabalina ,
What do you think about renaming `SurveyInfo` component to `SurveyOverview`, to align with the section heading?

```suggestion
import SurveyOverview from "../components/ResultsPageComponents/SurveyOverview/SurveyOverview";
```

Reply:
Happy with that Qing! ;-)